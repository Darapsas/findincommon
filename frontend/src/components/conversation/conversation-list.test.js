import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'
import ConversationList from './conversation-list'
import { MemoryRouter } from 'react-router-dom'

describe('<ConversationList />', () => {
  /*it('matches the snapshot', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <ConversationList />
        </MemoryRouter>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })*/
  it('placeholder', () => {
    expect()
  })
})
