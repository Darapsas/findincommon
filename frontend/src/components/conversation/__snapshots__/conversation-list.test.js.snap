// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConversationList /> matches the snapshot 1`] = `
<table
  className="table table-hover"
>
  <thead>
    <tr>
      <th
        scope="col"
      >
        #
      </th>
      <th
        scope="col"
      >
        Name
      </th>
      <th
        scope="col"
      >
        Created by
      </th>
      <th
        scope="col"
      />
      <th
        scope="col"
      />
      <th
        scope="col"
      />
    </tr>
  </thead>
  <tbody />
</table>
`;
